master/buildbot/steps/source/git.py

```diff
@@ -388,7 +386,10 @@ def _clone(self, shallowClone):
         command += [self.repourl, '.']
 
         if self.prog:
-            command.append('--progress')
+            if self.supportsProgress:
+                command.append('--progress')
+            else:
+                print("Git versions < 1.7.2 don't support progress")
```

Comment:
@rajdeepbharati: I think we should use `log.msg` here.

Reply:
Oops... I was wondering what to use